src/index.html

```diff
@@ -1,13 +1,67 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img 
+        src="images/imac.jpeg" 
+        alt="product-image" 
+        class="product-card__product-image"
+      >
+
+      <div class="product-card__product-name">
```

Comment:
```suggestion
    <div class="card" data-qa="card">
      <img 
        src="images/imac.jpeg" 
        alt="product-image" 
        class="card__image"
      >

      <div class="card__name">
```
Do not use such complicated names, you will only confuse yourself and other developers. In most cases, one word for the block and one for the element is enough

Reply:
Done